### ISSUE TITLE
Using $1D00 vs. $9D00 to document DOS3.3

### ISSUE THREAD

**Issue by TommyGH** [Michaelangel007/apple2_dos33#3]
I have an interesting idea for you to ponder. In your Github/Apple2_dos33 you are documenting DOS3.3 and its addressing based on a Master disk image and its initial load location ($1B00 relocator code and $1D00 for DOS proper) during the boot process.  Almost everyone tracing DOS or looking at DOS using AppleWin (or other emulator) will almost always be looking at the $9D00 final resting place of DOS after the boot process is complete, especially since (I think?) most DOS3.3 disks are slave disks.  

Have you thought about documenting Apple2_dos33 using the $9D00 addressing?  Or somehow allowing users to flip a switch and have everything magically change to the $9D00 addressing?   I know somewhere you mentioned you wanted to create a DOS3.3 symbol file for use with AppleWin, and certainly it (or a version of it) should be based on $9D00.

**Comment by Michaelangel007** [comment]
> Almost everyone tracing DOS or looking at DOS using AppleWin (or other emulator) will almost always be looking at the $9D00 final resting place of DOS after the boot process is complete,

Correct. Which is why I'll _also_ be providing AppleWin symbols at the final $9D00 range. :-)  I've added a new section 3.3 for where these will go.  It is trivial to add $8000 to all the addresses.

That is why I started this project in the first place! To get symbols for EVERY DOS 3.3 function and variable.

**Comment by Michaelangel007** [comment]
>  Or somehow allowing users to flip a switch and have everything magically change to the $9D00 addressing?

That's not a bad idea! I can put that in the top right in the "color/monochrome" option:

```
Address
(x) Master $1B00
( ) Slave $9B00
```

**Comment by Michaelangel007** [comment]
Internal notes for myself ...

* Prefix all AppleWin symbols: `DOS.` for DOS 3.3 -- we can still use `D32` (DOS 3.2) for the symbol tables if we ever do DOS 3.2
  * We have 11 characters _visible_ for symbol names in the debugger.
  * Longest DOS 3.3 symbol name is 8 characters
  * Which means 1 character will be hidden unless address/opcodes column is toggled off.
* Need to have symbol tables for:
  * [ ] Pronto DOS
  * [ ] Diversi DOS
* Markup `<a class='d'` variable declarations with `coords='16'` for 16 bytes. i.e.  `DB TABLE B64D:B65C`, etc.
  * [ ] Fix AppleWin's debugger to make equal sign optional for DB to be consistent with `SYM` and `ASC`. i.e. `DB TABLE = B64D:B65C` See: [AppleWin #449](https://github.com/AppleWin/AppleWin/issues/449)
* Markup `<a class='d'` variable declarations with `coords='-12'` for 2 byte text, i.e. `ASC VOLMES = B3AF:B3BA`
